packages/ra-materialui-treeview/src/TreeviewNode.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import KeyboardArrowDown from '@material-ui/icons/KeyboardArrowDown';
+
+import TreeviewNodeContent from './TreeviewNodeContent';
+
+const getRecordFromNode = ({ __children, __depth, ...record }) => record;
+
+const TreeviewNode = ({
+    basePath,
+    classes,
+    children,
+    node,
+    onChange,
+    resource,
+    theme,
+}) =>
+    node.__children.length > 0 ? (
+        <ListItem
+            button
+            classes={{
+                root: classes.root,
+            }}
+            dense
+            disableGutters
+            style={{ paddingLeft: theme.spacing.unit * 4 }}
```

Comment:
is this `style` and injected `theme` temporary?

Reply:
But this one doesn’t depend on props...

Reply:
woops ^^ good catch! It was before and I forgot to update it